components/Topic.js

```diff
@@ -1,7 +1,22 @@
 import React from "react";
 
-const Topic = () => {
-	return <div>Topic</div>;
+const Topic = ({ Topic, BackgroundColor, TextColor }) => {
+	return (
+		<div className='flex w-full h-full'>
+			<div
+				className='self-center box-border h-24 w-48 border-2 border-black rounded-3xl py-3'
```

Comment:
don't hard card the height and width, once you merge the PR for the setup, you can go from there